dracut/30ignition/coreos-teardown-initramfs-network.sh

```diff
@@ -2,7 +2,8 @@
 # -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
 # ex: ts=8 sw=4 sts=4 et filetype=sh
 
-set -euo pipefail
```

Comment:
Ouuff. What issues were you hitting with `set -euo pipefail` left on? Can we just turn it on after importing the library?

Reply:
Unset variables, commands that error, etc. We'd have to turn it off/on when we import the library and also each time we called a function in the library. Believe me I started doing it that way but declared bankruptcy. It's ugly.

Reply:
Ahhh. Hmm, one thing we could do is have a wrapper function like:

```sh
function dracut_fn() {
   set +euo pipefail
   "$@"
   set -euo pipefail
}
```

then call it as e.g. `dracut_fn ip_to_var ...`.

Anyway, IMO worth trying something along those lines if it works to maintain our sanity, but will leave it up to you.

Reply:
ok added a wrapper function!

Reply:
Very nice idea!

Reply:
I have this dream that someday 30 years from now, something will win and software engineers will see shell script like we see COBOL...